libs/books/data-access/src/lib/+state/reading-list.effects.spec.ts

```diff
@@ -42,4 +44,44 @@ describe('ToReadEffects', () => {
       httpMock.expectOne('/api/reading-list').flush([]);
     });
   });
+
+  describe('markBookAsFinished$', () => {
+    let item: ReadingListItem;
+    let finishedDate: string;
+
+    beforeAll(() => {
+      item = createReadingListItem('A');
+      finishedDate = new Date().toISOString();
+    });
+
+    it('should add a book to the reading list successfully', (done) => {
```

Comment:
Test case description is not matching with what it's doing inside. Please correct this.